core/src/main/java/io/micronaut/core/io/service/ServiceScanner.java

```diff
@@ -162,80 +106,10 @@ private Set<String> computeStandardServiceTypeNames(URL url) {
         return typeNames;
     }
 
-    private boolean isWebSphereClassLoader() {
-        return classLoader.getClass().getName().startsWith("com.ibm.ws.classloader");
-    }
-
-    private String buildResourceSearchPath() {
-        String path = "META-INF/micronaut/" + serviceName;
-
-        if (isWebSphereClassLoader()) {
```

Comment:
do we still need this websphere special case

Reply:
That was to look at the service directly now we should process the folder